pkg/tempopb/tempo.proto

```diff
@@ -91,9 +91,10 @@ message SpanSet {
 
 message Span {
   string spanID = 1;
-  uint64 startTimeUnixNano = 2;
-  uint64 durationNanos = 3;
-  repeated tempopb.common.v1.KeyValue attributes = 4;
+  string name = 2;
```

Comment:
I'm aware this is a breaking change, but AFAIK no one is consuming this API yet as protobuf. This is not a breaking change for JSON.